lib/lintbot.js

```diff
@@ -71,7 +70,7 @@ async function lintbot({
       retry,
       shouldRetry: err => err instanceof TimeoutError
         || (err instanceof FetchError && err.status >= 500 && err.status < 600),
-    });
```

Comment:
:stop_sign: **Lint Error** Missing semicolon.

```suggestion
    });
```

[semi](https://www.google.com/search?btnI&q=eslint%20rule%20semi)